src/app/forms.py

```diff
@@ -80,6 +80,7 @@ def __init__(self, *args, **kwargs):
     shortIdentifier = forms.CharField(label='Short identifier', max_length=25)
     sourceUrl = forms.CharField(label='Source / URL', required=False)
     osiApproved = forms.CharField(label="OSI Status", widget=forms.Select(choices=OSI_CHOICES))
+    exampleUrl = forms.CharField(label='Example Projects / URL', required=True)
```

Comment:
Should we use a TextField instead? For sourceUrl too. Although, the limit for CharField depends on the sql database we are using(255-65,535, goes to unlimited in some cases). I think we should be fine without it.
This change is not needed at all. Just pointing out for future reference.

Reply:
I couldn't find a TextField in forms - it looks like it is only in the DB model